src/workspace.coffee

```diff
@@ -589,7 +589,10 @@ class Workspace extends Model
 
   # Public: Register an opener for a uri.
   #
-  # An {TextEditor} will be used if no openers return a value.
+  # Atom loops through opener functions until one returns a value for a given uri.
```

Comment:
I think this line is confusing. You should probably mention that it does this looping you mention when someone uses the Atom API to open a file.